lib/elasticband/aggregation/nested.rb

```diff
@@ -1,18 +1,24 @@
 module Elasticband
   class Aggregation
     class Nested < Aggregation
-      attr_accessor :root_aggregation, :nested_aggregation
+      attr_accessor :root_aggregation, :nested_aggregations
 
-      def initialize(root_aggregation, nested_aggregation)
+      def initialize(root_aggregation, nested_aggregations)
         self.root_aggregation = root_aggregation
-        self.nested_aggregation = nested_aggregation
+        self.nested_aggregations = Array.wrap(nested_aggregations)
       end
 
       def to_h
         root_aggregation.to_h.tap do |h|
-          h[root_aggregation.name].merge!(aggs: nested_aggregation.to_h)
+          h[root_aggregation.name].merge!(aggs: nested_hash)
         end
       end
+
+      private
+
+      def nested_hash
+        nested_aggregations.each_with_object({}) { |a, h| h.merge!(a.to_h) }
```

Comment:
this `each` and `merge` is duplicated in: `lib/elasticband/aggregation.rb`. Is there a better way to reuse this code?